satpy/etc/readers/li_l2_nc.yaml

```
@@ -0,0 +1,193 @@
+    file_desc:
+      product_type: '2-AF'
+      variables: ['accumulation_offsets',
+                  'accumulation_start_times',
+                  'average_flash_qa',
+                  'flash_accumulation',
+                  'l1b_geolocation_warning',
+                  'l1b_radiometric_warning',
+                  'mtg_geos_projection',
+                  'x',
+                  'y']
+      variable_transforms:
+        accumulation_start_times:
+          seconds_to_datetime: "2000-01-01T00:00:00.000000"
+      swath_coordinates:
+        azimuth: 'x'
+        elevation: 'y'
+        projection: 'mtg_geos_projection'
+        variable_patterns: ['flash_accumulation']
+
+  li_l2_afr_nc:
+    file_reader: !!python/name:satpy.readers.li_l2_nc.LIL2NCFileHandler
+    file_patterns: ['{pflag}_{location_indicator},{data_designator},{mission_prefix}I{spacecraft_id}+LI-2-AFR-{subtype}-{coverage}-{subsetting}-{component1}-BODY-{component3}-{purpose}-{format}_{oflag}_{originator}_{processing_time:%Y%m%d%H%M%S}_{facility_or_tool}_{environment}_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{processing_mode}_{special_compression}_{disposition_mode}_{repeat_cycle_in_day:>04d}_{count_in_repeat_cycle:>04d}.nc']
+    file_desc:
+      product_type: '2-AFR'
+      variables: ['mtg_geos_projection',
+                  'accumulation_start_times',
+                  'accumulation_offsets',
+                  'x',
+                  'y',
+                  'flash_radiance',
+                  'l1b_geolocation_warning',
+                  'l1b_radiometric_warning',
+                  'average_flash_qa']
+      variable_transforms:
+        accumulation_start_times:
+          seconds_to_datetime: "2000-01-01T00:00:00.000000"
+      swath_coordinates:
+        azimuth: 'x'
+        elevation: 'y'
+        projection: 'mtg_geos_projection'
+        variable_patterns: ['flash_radiance']
+
+  li_l2_afa_nc:
+    file_reader: !!python/name:satpy.readers.li_l2_nc.LIL2NCFileHandler
+    file_patterns: ['{pflag}_{location_indicator},{data_designator},{mission_prefix}I{spacecraft_id}+LI-2-AFA-{subtype}-{coverage}-{subsetting}-{component1}-BODY-{component3}-{purpose}-{format}_{oflag}_{originator}_{processing_time:%Y%m%d%H%M%S}_{facility_or_tool}_{environment}_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{processing_mode}_{special_compression}_{disposition_mode}_{repeat_cycle_in_day:>04d}_{count_in_repeat_cycle:>04d}.nc']
+    file_desc:
+      product_type: '2-AFA'
+      variables: ['mtg_geos_projection',
+                   'accumulation_start_times',
+                   'accumulation_offsets',
+                   'x',
+                   'y',
+                   'accumulated_flash_area',
+                   'l1b_geolocation_warning',
+                   'l1b_radiometric_warning',
+                   'average_flash_qa']
```

Intendation is a bit off
```suggestion
      variables: ['mtg_geos_projection',
                  'accumulation_start_times',
                  'accumulation_offsets',
                  'x',
                  'y',
                  'accumulated_flash_area',
                  'l1b_geolocation_warning',
                  'l1b_radiometric_warning',
                  'average_flash_qa']
```